accounts/src/main.rs

```diff
@@ -0,0 +1,20 @@
+use tonic::transport::Server;
+
+mod error;
+mod proto;
+mod service;
+
+/// Start the RPC service.
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    Server::builder()
+        .add_service(proto::accounts_server::AccountsServer::new(
+            service::AccountsService::default(),
+        ))
+        .serve("[::1]:50051".parse().unwrap())
+        .await?;
+
+    println!("RPC service has been started.");
```

Comment:
Just a general thing, but perhaps it's nice to use a proper logger and the `log` crate instead of println

Reply:
There isn't much to log yet, but it's probably a good habit to get in, I've changed this.